torch/ao/quantization/quantize_fx.py

```diff
@@ -196,12 +198,27 @@ def _prepare_fx(
     """
     if prepare_custom_config_dict is None:
         prepare_custom_config_dict = {}
-    if equalization_qconfig_dict is None:
-        equalization_qconfig_dict = {}
+    if equalization_quantization_config is None:
```

Comment:
I think equalization_quantization_config is a bit weird, maybe just `equalization_config`?